src/api/spec/features/webui/image_templates_spec.rb

```diff
@@ -1,32 +1,56 @@
 require "browser_helper"
+require 'feature/testing'
```

Comment:
No worries, we can have it inside the `rails_helper`, it just adds 3 methods to the class.